### ISSUE TITLE
processes of draw.io still running after close GUI on app image.

### ISSUE THREAD

**Issue by ibexnet** [jgraph/drawio-desktop#1487]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

**Describe the bug**
When I close drawio application, the GUI interface close without errors but application's processes still running:
```
> ps aux | grep drawio
user      218715 89.5  0.9 1175406300 156864 ?   Rl   20:43   1:11 /tmp/.mount_drawioBSepfT/drawio
user      218719  4.9  0.0  12044  1760 ?        Ssl  20:43   0:03 /home/user/Applications/drawio/drawio
user      218723  0.0  0.3 33788056 50304 ?      S    20:43   0:00 /tmp/.mount_drawioBSepfT/drawio --type=zygote --no-zygote-sandbox
user      218724  0.0  0.3 33788044 50432 ?      S    20:43   0:00 /tmp/.mount_drawioBSepfT/drawio --type=zygote
user      218726  0.0  0.0 33788072 12308 ?      S    20:43   0:00 /tmp/.mount_drawioBSepfT/drawio --type=zygote
user      218982  0.0  0.0   9540  2560 pts/0    S+   20:44   0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn --exclude-dir=.idea --exclude-dir=.tox drawio
```
After several minutes, the process '/tmp/.mount_drawioBSepfT/drawio', take 100% of CPU usage.
I must manually kill that process or it run forever.

**To Reproduce**
1. Open draw.io
2. Click on button for close the window of GUI.
3. If I look on running processes, drowio processes still running

**Expected behavior**
After I closed the program GUI window I expect that no processes about application still running

**Screenshots**

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 22.0.2 (app image)

**Desktop (please complete the following information):**
 - OS: Linux, Ubuntu 22.04.3

**Additional context**

**Comment by m-mohamedin** [comment]
Unfortunately, we couldn't reproduce. Please share more details. Does it still occur with the latest version

**Comment by ibexnet** [comment]
Hi @m-mohamedin, seem that with the latest version (22.1.15) the behavior is changed.
Now when I close the windows GUI no running process about drawio persist
But I saw that the drawio process close unexpectly with segmentation fault:
```
╭─user at host in ~/Applications/drawio
╰─○./drawio                     
17:46:09.928 › Checking for update
File INI:/usr/local/lib/ciepki.ini
Inizio Sessione - versione: Sep 15 2022 17:10:03Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
Lib log level: 2
17:46:11.553 › Update for version 22.1.15 is not available (latest version: 22.1.15, downgrade is disallowed).
[1]    844064 trace trap (core dumped)  ./drawio
╭─user at host in ~/Applications/drawio
╰─○ echo $? 
133

```

anyway the program seem work correctly and the boring behavior of processes that hang and use all the CPU is gone so somehow the issue was fixed : )


thanks

**Comment by m-mohamedin** [comment]
Great, I'll close
Thanks for the report
